Doc/howto/annotations.rst

```diff
@@ -57,6 +57,10 @@ Accessing The Annotations Dict Of An Object In Python 3.10 And Newer
   newer is to call :func:`getattr` with three arguments,
   for example ``getattr(o, '__annotations__', None)``.
 
+  Starting from Python 3.10, accessing the annotations
+  of a class will not give anymore the ones of its base
+  classes.
```

Comment:
I would say something like this:

Before Python 3.10, accessing ``__annotations__`` on a class that defines no annotations but that has a parent class with annotations would return the parent's ``__annotations__``. In Python 3.10 and newer, the child class's annotations will be an empty dict instead.

Reply:
The section in which the paragraph is located is explicitely about Python 3.10+, but I like the idea of being more explicit.
What do you think about:

Starting from Python 3.10 and newer, classes do not inherit anymore the `__annotations__` of their parent classes, so, accessing a class that does not define `__annotations__` will always return an empty dict.

Reply:
I think @JelleZijlstra's version is clearer. Even though the section is already about 3.10+, it is still clearer (and not out of place) to first explain the old behavior and then contrast the new behavior.